Configuration/Eras/python/Era_Phase2C1_timing_cff.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Modifier_run2_common_cff import run2_common
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
+from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
+
+Phase2C1_timing = cms.ModifierChain(run2_common, phase2_common, phase2_tracker, trackingPhase2PU140, phase2_muon, run3_GEM, phase2_timing)
```

Comment:
If the point is to be `Phase2C1` + `phase2_timing` always, how about `Phase2C1 = cms.ModifierChain(Phase2C1, phase2_timing)`? Then all future changes to `Phase2C1` would be automatically included here.

(same comment for `Phase2C2_timing` below)